app/presenters/check_answers/case_disposal_card.rb

```diff
@@ -9,5 +9,40 @@ def initialize(claim)
       @group = 'about_case'
       @section = 'case_disposal'
     end
+
+    def find_key_by_value(value_to_find)
+      choices = {
+        guilty_pleas: PleaOptions::GUILTY_OPTIONS,
+        not_guilty_pleas: PleaOptions::NOT_GUILTY_OPTIONS,
+      }
+      plea_to_find = PleaOptions.new(value_to_find)
+      choices.each do |key, options|
+        return key if options.include?(plea_to_find)
+      end
+      nil # Return nil if the value is not found in any of the choices
+    end
```

Comment:
This is really a private method of the class, but looks to have been made public top to help/allow testing - have we thought about testing this without it being public?

Reply:
yes, thought about and tried. couldn't find a way. If you have any suggestions we can have a 5mins chat to figure it out.